simplq/src/components/common/Loading/Loading.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropagateLoader from 'react-spinners/PropagateLoader';
+import styles from './Loading.module.scss';
+
+export default ({ children, actionStatus }) => {
+  if (!actionStatus || actionStatus === 'pending') {
+    return (
+      <div className={styles.main}>
+        <PropagateLoader color="#3a3768" />
+      </div>
+    );
+  }
+
+  if (!actionStatus || actionStatus === 'rejected') {
+    return (
+      <div className={styles.main}>
+        An unknown error occured. Please look at the console log for more nfo.
```

Comment:
Lets replace with this:

An unknown error occurred. Our team is looking into this. You can also reach out to us [here](https://github.com/SimplQ/simplQ-frontend/issues/new/choose) if you want to follow up on this.

Reply:
Also lets put this behaviour behind a flag `displayErrors` which would defaults to true (🤔 ) as it will otherwise upset tiny spinners.

Reply:
![image](https://user-images.githubusercontent.com/10821730/107263169-6ff03900-6a0f-11eb-899f-0f9f7565d78f.png)

A bit too long 😄

> Also lets put this behaviour behind a flag displayErrors which would defaults to true (🤔 ) as it will otherwise upset tiny spinners.

I don't think anything is necessary here. It will just help to catch unhandled errors.

Reply:
Right. I think unhandled errors can be silently passed to sentry without polluting the spinner component.